tests/Unit/Framework/Pypp.hpp

```diff
@@ -21,262 +22,309 @@
 /// Contains all functions for calling python from C++
 namespace pypp {
 namespace detail {
+template <typename R>
```

Comment:
I'd prefer a more explicit template parameter (also there are a few others like that in this file, but for some reason `R` seems more confusing to me than `T`)

Reply:
Ah, `R` is short for `ReturnType`. How about I add a commit that makes the change in the entire file at the end?

Reply:
Sounds great